cassandra-migration/src/main/resources/db/migration/cassandra/V1.27_cassandra.cql

```diff
@@ -0,0 +1,7 @@
+cqlsh> 

```

Comment:
Only CQL queries should be part of this file. I see "cqlsh>" prompts copy pasted here which is incorrect.